crates/forge/src/mutation/mutant.rs

```diff
@@ -0,0 +1,350 @@
+// Generate mutants then run tests (reuse the whole unit test flow for now, including compilation to
+// select mutants) Use Solar:
+use solar_parse::ast::{BinOpKind, Expr, ExprKind, LitKind, Span, UnOpKind};
+use std::path::PathBuf;
+
+use super::visitor::AssignVarTypes;
+
+/// Wraps an unary operator mutated, to easily store pre/post-fix op swaps
+#[derive(Debug)]
+pub struct UnaryOpMutated {
+    /// String containing the whole new expression (operator and its target)
+    /// eg `a++`
+    new_expression: String,
+
+    /// The underlying operator used by this mutant
+    resulting_op_kind: UnOpKind,
+}
+
+impl UnaryOpMutated {
+    pub fn new(new_expression: String, resulting_op_kind: UnOpKind) -> Self {
+        UnaryOpMutated { new_expression, resulting_op_kind }
+    }
+}
+
+impl ToString for UnaryOpMutated {
+    fn to_string(&self) -> String {
+        self.new_expression.clone()
+    }
+}
+
+#[derive(Debug)]
+pub enum MutationType {
```

Comment:
context: these are the mutators used by Gambit, in theory (in practice, some are not implemented, or not defined - see https://github.com/Certora/gambit/blob/bf7ab3c91c47a10dcf272380b6406f0404f3b5d1/src/mutation.rs#L323 for instance)